tools/launch.ts

```diff
@@ -118,6 +118,11 @@ const parachains: { [name: string]: ParachainConfig } = {
     chain: "moonbase-local",
     docker: "purestake/moonbeam:v0.18.0",
   },
+  "moonbase-0.18.1": {
```

Comment:
We should probably only store 1 config per minor version (0.18)

Reply:
So we would bump the above `0.18.0` to `0.18.1`, then?

Reply:
Yes we can bump the above `0.18.0` to `0.18.1`, anyway this branch is not intended to be merged on master.

Reply:
done